cdc/sorter/unified/unified_sorter.go

```diff
@@ -34,9 +34,8 @@ const (
 	heapCollectChSize = 128 // this should be not be too small, to guarantee IO concurrency
 )
 
-// UnifiedSorter provides both sorting in memory and in file. Memory pressure is used to determine which one to use.
-//nolint:revive
-type UnifiedSorter struct {
+// Sorter provides both sorting in memory and in file. Memory pressure is used to determine which one to use.
+type Sorter struct {
```

Comment:
`Sorter` might be too generic a name and might hinder readability?

Reply:
It's easy to distinguish, because our pkg is `unified`.

unified.Sorter
memory.Sorter